### ISSUE TITLE
Windows release crashes

### ISSUE THREAD

**Issue by gnunicorn** [acterglobal/a3#1922]
reported by the microsoft certification team, confirmed on our local laptop: the release exe is crashing on start without further information. Just the screen appearing for a few seconds, no logo, nothing, then it disappears entirely.

**Comment by gnunicorn** [comment]
so. [I have a fix](https://github.com/acterglobal/a3/pull/1857/commits/866fb47b3fe1801d41c0b823ac914fee12793dc5). For some reason we are building against the latest msvc libs but then flutter puts different ones into the release folder. Es also reported in [this mediakit issue report](https://github.com/media-kit/media-kit/issues/842).

Which for a long time was fine, not recommended - recommended [was to always ship with the latest you've built against](https://docs.flutter.dev/platform-integration/windows/building#distributing-windows-apps) - but fine. but in a recent update (well, 2022) of the libs they changed some behaviors and if you relied on it working with old ones, you were at fault. because the old ones misbehaved and you didn't ship the latest ones. With that patch we are shipping the redstributions libs we built against, which should always be the latest the github runner provides.

**Comment by gnunicorn** [comment]
Our msix run was also rejected because it crashed again. To ensure this wasn't an unrelated problem I tried the exe-release we had build (and that worked) on a fresh account on windows and find it works fine. Next I pulled the `msix` we had sent to store, extracted it contents with the microsoft msix packaging tool and find that this, too, crashes with the same `do_mtxlock` problem. Copying over the redistribution libs from the exe-folder fixes it and it runs fine.

Turns out the msix package tries to be smart and [ships with its own version of the msix binaries](https://github.com/YehudaKremer/msix/issues/272), which are now a mismatch to what is needed. We don't want that, we want it to use the ones we are already copying over into the Release directory. [Luckily that is what a fork of the msix package already does and we are trying out now](https://github.com/YehudaKremer/msix/issues/272#issuecomment-2181634105).
